pypsdm/io/utils.py

```diff
@@ -164,3 +182,16 @@ def bool_converter(maybe_bool):
         return maybe_bool.lower() == "true"
     else:
         raise ValueError("Cannot convert to bool: " + str(maybe_bool))
+
+
+def normaldistribution(dist_params):
+    normal_variant = normalvariate(dist_params[0], dist_params[1])
+    return max(min(normal_variant, dist_params[3]), dist_params[2])
```

Comment:
We should add some documentation to this method to specify the order of the parameters.